commcare_connect/connect_id_client/main.py

```diff
@@ -94,6 +95,7 @@ def get_user_otp(phone_number):
                 "that the user has started their device seating process.",
             )
         else:
+            sentry_sdk.capture_message(message=f"ConnectID client error: {str(e)}", level="error")
```

Comment:
nit: I think it makes more sense to add this to the view where we catch the exception so that it logs the actual exception class to sentry, like a normal exception there rather than a custom message

Reply:
Somehow I don't like my own solution here: https://github.com/dimagi/commcare-connect/commit/c87b6f2d1e19819e426136256a6c1b5f62145769

I do think exceptions should be logged further up-stream, closer to the source (i.e. inside `_make_request` where we raise for status). I can create another PR for that if you agree in the interest of not getting side-tracked in this PR.

Reply:
I think since we aren't catching the exception this has become a moot point (and explicit sending is no longer necessary.